### PR TITLE
Update MoPub dependency to 5.13.1

### PR DESCRIPTION
**Pull request by TurpIF**
JIRA: EE-1179

### CODE REVIEW

File: publisher-sdk-tests/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -3,6 +3,17 @@
   xmlns:tools="http://schemas.android.com/tools"
   package="com.criteo.publisher.tests">
 
+  <!-- MoPub requires minSdk=19, while the SDK had minSdk=16 -->
```
Comment (Amokrane) [inline publisher-sdk-tests/src/main/AndroidManifest.xml:6]: Can you add a `See https://developer.android.com/studio/build/manifest-merge` comment please?

File: app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -1,9 +1,21 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:tools="http://schemas.android.com/tools"
   package="com.criteo.pubsdk_android">
 
   <uses-permission android:name="android.permission.INTERNET"/>
 
+  <!-- MoPub requires minSdk=19, while the SDK had minSdk=16 -->
```
Comment (Amokrane) [inline app/src/main/AndroidManifest.xml:8]: Can you add a `See https://developer.android.com/studio/build/manifest-merge` comment please?

File: publisher-sdk-tests/src/main/AndroidManifest.xml (hunk 1 above)
Comment (TurpIF) [reply publisher-sdk-tests/src/main/AndroidManifest.xml]: done

File: app/src/main/AndroidManifest.xml (hunk 2 above)
Comment (TurpIF) [reply app/src/main/AndroidManifest.xml]: done

**Pull request merged**